.travis.yml

```diff
@@ -1,19 +1,28 @@
 ---
```

Comment:
It might be possible that we had to add the dist param [1], maybe that's why CI isn't running...

1 https://github.com/idealista/mysql_role/blob/master/.travis.yml#L2